casper/src/main/resources/NonNegativeNumber.rho

```diff
@@ -17,7 +17,7 @@ new NonNegativeNumber, rs(`rho:registry:insertSigned:ed25519`), uriOut in {
       contract this(@"add", @x, success) = {
         if (x >= 0) {
           for(@v <- valueStore){
-            if (v + x > v) {
+            if (v + x >= v) {
```

Comment:
Yup! I think such test should fail before your change

Reply:
👍

Reply:
So, just to be clear, I'd introduce a test prior to this commit that would fail for all refs prior to this commit. If all commits in the history should compile, shouldn't we try our best to have all commits pass tests as well?

Reply:
You would not :) You'd introduce the test with the fix, or just after it.

Reply:
> I think such test should fail before your change

This was just supposed to mean "that's a valid test for your change"